semantic-core/test/Spec.hs

```diff
@@ -93,10 +95,10 @@ parserSpecs = testGroup "Parsing: simple specs"
   , testCase "quoted names" assert_quoted_name_parse
   ]
 
-assert_roundtrips :: File (Term (Ann :+: Core) Name) -> Assertion
-assert_roundtrips (File _ core) = case parseEither Parse.core (showCore (stripAnnotations core)) of
-  Right v -> stripAnnotations v @?= stripAnnotations core
-  Left e -> assertFailure e
+assert_roundtrips :: File (Term Core Name) -> Assertion
+assert_roundtrips (File _ _ core) = case parseEither Parse.core (showCore core) of
+  Right v -> v @?= core
+  Left  e -> assertFailure e
```

Comment:
Our parser never produces annotations, and is polymorphic in the signature type, so we parse directly into `Term Core Name` instead of `Term (Ann Span :+: Core) Name` and thus avoid having to `stripAnnotations`.